### PR TITLE
Add grouping by name

### PR DESCRIPTION
**Pull request by Domino987**
This PR adds a toggle to group by name, so that multiple items per row are possible.

By adding the groupName to the tasks, they get grouped into one row.

Let me know what you think :)

It turns this:
image
into 
image
by providing
```
groupName: 'Task'
```

### CODE REVIEW

**Comment by Domino987** [comment]
What do you think @guiqui

**Review by rhfauzi — APPROVED** [review]

**Comment by sodhanii** [comment]
Hi, can we merge this PR in master branch?

File: src/lib/TimeLine.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ class TimeLine extends Component {
   onFinishCreateLink = (task, position) => {
     console.log(`End Link ${task}`);
     if (this.props.onCreateLink && task &&
-      this.state.taskToCreate &&this.state.taskToCreate.task.id!=task.id) {
+      this.state.taskToCreate && this.state.taskToCreate.task.id != task.id) {
```
Comment (bikkimahato) [inline src/lib/TimeLine.js:267]: Use strict inequality !==

File: src/lib/helpers/registry/Registry.js [hunk 2]
```diff
@@ -1,47 +1,63 @@
...
+            this.createAddTo(end, this.link, value, i)
         }
     }
-    createAddTo(id,list,value,index){
+    createAddTo(id, list, value, index) {
         if (!list[id])
-            list[id]=[]
-        if (list[id].indexOf(value)==-1)
+            list[id] = []
+        if (list[id].indexOf(value) == -1)
```
Comment (bikkimahato) [inline src/lib/helpers/registry/Registry.js:36]: Strict equality operator should be used ===

**Comment by guiqui** [comment]
> What do you think @guiqui

Thank you for your contribution! Really appreciate it .

**Pull request merged**